sdk/python/examples/quickstart/Quickstart.ipynb

```diff
@@ -0,0 +1,239 @@
+    "\n",
+    "# location features\n",
+    "df.loc[:, 'distance_haversine'] = haversine_array(df['pickup_latitude'].values, df['pickup_longitude'].values, df['dropoff_latitude'].values, df['dropoff_longitude'].values)\n",
+    "df.loc[:, 'distance_dummy_manhattan'] =  dummy_manhattan_distance(df['pickup_latitude'].values, df['pickup_longitude'].values, df['dropoff_latitude'].values, df['dropoff_longitude'].values)\n",
+    "df.loc[:, 'direction'] = bearing_array(df['pickup_latitude'].values, df['pickup_longitude'].values, df['dropoff_latitude'].values, df['dropoff_longitude'].values)\n",
+    "\n",
+    "# time features\n",
+    "df['month'] = df['pickup_datetime'].dt.month\n",
+    "df['day_of_month'] = df['pickup_datetime'].dt.day\n",
+    "df['hour'] = df['pickup_datetime'].dt.hour\n",
+    "df['day_of_week'] = df['pickup_datetime'].dt.dayofweek\n",
+    "\n",
+    "# one hot encoding\n",
+    "vendor = pd.get_dummies(df['vendor_id'], prefix='vi', prefix_sep='_')\n",
+    "store_and_fwd_flag = pd.get_dummies(df['store_and_fwd_flag'], prefix='sf', prefix_sep='_')\n",
+    "\n",
+    "df = df.drop(['trip_duration','vendor_id','passenger_count','store_and_fwd_flag', 'dropoff_datetime',\n",
+    "           'pickup_longitude','pickup_latitude','dropoff_longitude','dropoff_latitude'],axis = 1)\n",
+    "df.columns = ['ride'] + list(df.columns[1:])\n",
+    "df_complete = pd.concat([df, vendor, store_and_fwd_flag], axis=1)\n",
+    "df_complete.columns = [col.lower() for col in df_complete.columns]\n",
+    "df_complete.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Ingesting into Feast"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "FEAST_CORE_URL = 'localhost:6565'\n",
+    "FEAST_SERVING_URL = 'localhost:6566'\n",
+    "STAGING_LOCATION = 'gs://feast-bucket/staging'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Now that we have finished creating our features, we ingest them into feast\n",
+    "\n",
+    "# Initialise client\n",
+    "fs = Client(core_url=FEAST_CORE_URL, verbose=True)\n",
+    "\n",
+    "# Create importer\n",
+    "importer = Importer.from_df(df_complete, \n",
+    "                           entity='ride', \n",
+    "                           granularity=Granularity.NONE,\n",
+    "                           owner='user@website.com',  \n",
+    "                           staging_location=STAGING_LOCATION,\n",
+    "                           id_column='ride', \n",
+    "                           timestamp_column='pickup_datetime')\n",
+    "\n",
+    "# Update feature and entity metadata\n",
+    "serving_ds=Datastore(id='REDIS1').spec\n",
+    "warehouse_ds=Datastore(id='NOOP').spec\n",
```

Comment:
Why do we need `spec`? Shouldn't specifications only be when we want to serialize the objects for storage?

Reply:
Also, can't we make this experience nicer? Have the importer define a default `datastore`? or at least set the `datastores` when reading in the actual dataframe?